client/dist/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Cap BnB</title>
+    <link rel="stylesheet" href="./styles.css" type="text/css">
+    <script src="https://code.jquery.com/jquery-3.4.1.js"
+      integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU="
+      crossorigin="anonymous"></script>
+      <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
```

Comment:
Are either of these being used? You can `npm install jquery` and `import` it at the top of any React file you need to use it in.

Reply:
As for the Montserrat font, aren't we using AirBnB-Cereal?

Reply:
@FrancoisArago 
I wasn't using AirBnB-Cereal because I tried it when first setting up views and it didn't end up looking like the font that was on the AirBnB site. I pulled these fonts from the AirBnB site. Does Cereal work in the way you expect it to work? Did you try using it in yours? Maybe I wasn't using it correctly. It turned out to be a serif font when displayed in my client. 

I'll be using jQuery for my AJAX request. But as for the font, I'm interested in your feedback about the use of Cereal.

Reply:
@FrancoisArago - disregard my comment about AirBnB cereal - I'm using it and adding a sans-serif attribute to the font and it seems to look good now.